public-api/src/main/java/org/pmiops/workbench/publicapi/CdrVersionsController.java

```diff
@@ -0,0 +1,75 @@
+package org.pmiops.workbench.publicapi;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.ImmutableSet;
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import javax.inject.Provider;
+import org.pmiops.workbench.config.WorkbenchConfig;
+import org.pmiops.workbench.db.dao.CdrVersionDao;
+import org.pmiops.workbench.db.model.CdrVersion;
+import org.pmiops.workbench.db.model.CommonStorageEnums;
+import org.pmiops.workbench.exceptions.ServerErrorException;
+import org.pmiops.workbench.model.CdrVersionListResponse;
+import org.pmiops.workbench.model.DataAccessLevel;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class CdrVersionsController implements CdrVersionsApiDelegate {
```

Comment:
Is it not possible for this to live in common-api?

Reply:
The implementation is slightly different; this one doesn't depend on an authenticated user (we don't have one). I could try to refactor out common code if you'd like into a base class in common-api... given how simple the logic is, I wasn't sure it was worth the trouble.

Reply:
Yeah. Not worth the trouble now.

Reply:
We will be able to have a common controller though for common endpoints that are exactly the same right ? I can see a lot of the stuff in public being common eventually

Reply:
If they were exactly identical, sure, it could live in common. And when they differ slightly (as in this case), we have the option of making a base controller class in common-api with common code, and subclasses in api and public-api.